cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -322,6 +322,13 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
         }
     }
 
+    private fun isOverloadedOperator(ex: Expression, operatorCode: String): Boolean {
```

Comment:
a) some small kdoc would be nice
b) I would prefer that the name of the function reflects that this only checks for overloaded member expressions.